tasks/main.yml

```diff
@@ -9,7 +9,7 @@
 - name: Download and untar the guacamole-composition tarball
   ansible.builtin.unarchive:
     src: >
-      https://api.github.com/repos/cisagov/guacamole-composition/tarball/develop
+      https://api.github.com/repos/cisagov/guacamole-composition/tarball/v0.1.4-rc.5
```

Comment:
I know the description mentions reverting this, but are you reverting to `develop` or `v0.1.4`?

Reply:
The latter.  See commit 930c381.

Reply:
Excellent. Exactly what I was hoping to hear.